puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/tests/testForeman.groovy

```diff
@@ -0,0 +1,135 @@
+node('fast') {
+
+    stage('Setup Git Repos') {
+
+        git url: "https://github.com/theforeman/foreman", branch: "develop"
+
+    }
+
+    stage("Setup RVM") {
+
+        withRVM([
+            'rvm gemset empty --force',
+            'gem update --no-ri --no-rdoc',
+            'gem install bundler --no-ri --no-rdoc'
+        ])
+
+    }
+
+    stage('Configure Environment') {
+
+        try {
+            sh "cp config/settings.yaml.example config/settings.yaml"
```

Comment:
Sorry, I don't entirely follow what you are asking for. This step does create `settings.yaml` via the copy.